edufinder/edufinder/rest_api/migrations/0007_auto_20201104_1132.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.2 on 2020-11-04 11:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rest_api', '0006_useranswer_education'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='question',
+            old_name='question',
+            new_name='da',
+        ),
+        migrations.AddField(
+            model_name='question',
+            name='en',
+            field=models.CharField(default='asd', max_length=200),
```

Comment:
Default to `asd` may be fine as a temporary value - but consider updating [testdata.json](https://github.com/sw708e20/edufinder/blob/fae6380071491515b73dc04878b1de15fc875539/edufinder/edufinder/rest_api/fixtures/testdata.json#L24) to add the translations.